articles/reactive-repeat.md

```diff
@@ -4,3 +4,10 @@ title: Repeat
 ---
 
 ![Marble diagram](~/images/reactive-repeat.svg)
+
+`Repeat` reacts to successful termination by disposing the previous subscription and resubscribing to the source sequence. Elements received from all of the subscriptions are forwarded to the result sequence.
+
+> [!Warning]
+> If any of the subscriptions terminates exceptionally, the result sequence will also terminate exceptionally, and no further resubscriptions will be made.
```

Comment:
Add a warning when using Repeat and branched workflows. Repeat will only restart the sequence when all subscribers close, right?

Reply:
Technically this isn't a problem with `Repeat`, but rather with the publish behavior of branches. There is already a warning in the main conceptual documentation here: https://bonsai-rx.org/docs/articles/subjects.html#branching-subjects

However, it is true that for many people they will hit the problem when dealing with `Repeat` and its variants. Maybe it is enough to have a warning pointing to that discussion?

Reply:
Yup, I agree that a mention of the discussion should suffice.